playbooks/common/openshift-master/config.yml

```diff
@@ -117,6 +117,7 @@
                                      | oo_collect('openshift.common.hostname')
                                      | default(none, true) }}"
     openshift_master_hosts: "{{ groups.oo_masters_to_config }}"
+    r_etcd_common_etcd_runtime: "{{ openshift.common.etcd_runtime }}"
```

Comment:
Would be better to introduce `r_openshift_master_etcd_runtime` and propagate it to the `etcd_common` role. However, given the dependency chain `openshift_master` -> `openshift_etcd_client_certificates` -> `etcd_client_certificates` -> `etcd_common`, the roles needs some refactoring before we will have super cool handling of variables or indirect dependencies.